scripts/wait_for_couch.sh

```diff
@@ -0,0 +1,6 @@
+count=0
```

Comment:
can you either rename this to make it clear this is only for the e2e couchdb (because of 4984) or have the port passed as a parameter. I worry that months from now it will be confusing without one of those two changes

Reply:
Agree, this would be my only comment as well.
Maybe we could put our `e2e` stuff under a single folder, so far it would be this script + `scripts/e2e-servers.js`.